UnityProject/Assets/Scripts/Player/PlayerSync.Server.cs

```diff
@@ -472,11 +472,11 @@ private void TryUpdateServerTarget()
 		lastDirectionServer = Vector2Int.RoundToInt(newPos - oldPos);
 		ServerState = nextState;
 		//In case positions already match
-		TryNotifyPlayers();
```

Comment:
why did you remove this one? removing it might make other players receive position updates later

Reply:
since this is at the start of the movement, it instantly gets cancelled out because the positions are the same

Reply:
you probably meant `positions are no longer the same`? since `ServerState` was assigned with `nextState` and `ServerPositionsMatch` is no longer true – yeah, that would make sense
otherwise `SyncMatrix()` call from `TryNotifyPlayers()` would be executed and freezing wouldn't be an issue in the first place